src/components/search/searchfields.js

```diff
@@ -1,10 +1,20 @@
-define(['layoutManager', 'globalize', 'require', 'events', 'browser', 'alphaPicker', 'emby-input', 'flexStyles', 'material-icons', 'css!./searchfields'], function (layoutManager, globalize, require, events, browser, AlphaPicker) {
-    'use strict';
+import layoutManager from 'layoutManager';
+import globalize from 'globalize';
+import require from 'require';
+import events from 'events';
+import browser from 'browser';
+import AlphaPicker from 'alphaPicker';
+import 'emby-input';
+import 'flexStyles';
+import 'material-icons';
+import 'css!./searchfields';
+
+/* eslint-disable indent*/
```

Comment:
more weird indentation on these comments

Reply:
Thanks, I'll fix it